TACC/Frontera-Portal
# Task/fp 783 onboarding get help link

## Overview: ##

Add "Get Help" link to onboarding

## PR Status: ##

* [x] Ready (but some TODOs from current review)

## Related Jira tickets: ##

* [FP-783](https://jira.tacc.utexas.edu/browse/FP-783)

## Summary of Changes: ##
* Add "Get Help" link to onboarding
* Refactor Ticket creation modal so that it can be opened through redux-saga

## Testing Steps: ##
1. See _"Get Help"_ link on onboarding page (`/workbench/onboarding`)
2. Test create ticket modal by clicking _"Get Help"_ link
3.  Also, ensure ticket creation modal on Dashboard still works properly (`/workbench/dashboard/tickets/create`)
4.  Also, ensure ticket creation modal on Dashboard still works properly (`/workbench/dashboard/tickets/create`)
5.  Also, ensure that creating a ticket about software license access still works properly  (https://cep.dev/workbench/account.   "Request Allocation") 

## UI Photos:

![Screen Shot 2020-11-18 at 8 53 25 AM](https://user-images.githubusercontent.com/8287580/99546433-031f5280-297c-11eb-8e2c-6456439c2340.png)
![Screen Shot 2020-11-18 at 8 53 45 AM](https://user-images.githubusercontent.com/8287580/99546424-01558f00-297c-11eb-9504-441b4e54877e.png)

## Notes: ##

TODO:
- [x] ~check ticket creation notice color as it seems to be wrong~ going to create a seperate ticket/pr to use _common/Message instead of bootstrap Alert
- [x] consider making dashboard use of modal also through a dispatch
- [x] Add "Onboarding" to the default subject for Get Help from onboarding.
